Backend/dev/test-tool/context-api/server.py

```
@@ -3029,6 +3029,15 @@ def _proxy(self, method):
         else:
             return False
 
+        _xpt = self.headers.get("X-Proxy-Target", "").strip()
+        if _xpt:
+            try:
+                _xp = urlparse(_xpt)
+                if _xp.scheme in ("http", "https") and _xp.netloc:
+                    target_base = f"{_xp.scheme}://{_xp.netloc}"
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)